### PR TITLE
Plans: show domain feature notice in stats insights.

### PR DESCRIPTION
**Pull request by mtias**
Rendered at the bottom of the full-width blocks:

image

### CODE REVIEW

**Comment by gwwar** [comment]
This feels pretty close to a domain tip: p3fqKv-2de-p2 We may want to consider updating the copy in the future, or removing this nudge if we create a new domain tip component.

File: client/my-sites/stats/stats-insights/index.jsx [hunk 1]
```diff
@@ -84,6 +85,12 @@ export default React.createClass( {
 					/>
 					<AllTime allTimeList={ allTimeList } />
 					<MostPopular insightsList={ insightsList } />
+					<UpgradeNudge
```
Comment (gwwar) [inline client/my-sites/stats/stats-insights/index.jsx]: Does this not need to be behind our generic nudge abtest?

File: client/my-sites/stats/stats-insights/index.jsx [hunk 2]
```diff
@@ -13,11 +13,12 @@ import Comments from '../stats-comments';
 import Followers from '../stats-followers';
 import PostingActivity from '../post-trends';
 import TodaysStats from '../stats-site-overview';
-import SiteOverviewPlaceholder from 'my-sites/stats/stats-overview-placeholder';
 import StatsModule from '../stats-module';
 import statsStrings from '../stats-strings';
 import MostPopular from 'my-sites/stats/most-popular';
 import LatestPostSummary from '../post-performance';
+import UpgradeNudge from 'my-sites/upgrade-nudge';
+import { CUSTOM_DOMAIN } from 'lib/plans/constants';
```
Comment (gwwar) [inline client/my-sites/stats/stats-insights/index.jsx]: Do you mean `FEATURE_CUSTOM_DOMAIN`? 

I'm seeing false being recorded for cta_feature:
screen shot 2016-04-28 at 3 17 47 pm

File: client/my-sites/stats/stats-insights/index.jsx (hunk 1 above)
Comment (mtias) [inline client/my-sites/stats/stats-insights/index.jsx]: That's going to be done within the component in #5073

File: client/my-sites/stats/stats-chart-tabs/index.jsx [hunk 3]
```diff
@@ -213,25 +209,6 @@ export default React.createClass( {
 		return chartData;
 	},
 
-	renderNudge: function() {
-		if ( abtest( 'statsTabsLikesNudge' ) === 'dataInformedBelowChart' ) {
-			return (
-				<UpgradeNudge
-					title={ this.translate( 'Sites with Premium get 31% more likes' ) }
```
Comment (artpi) [inline client/my-sites/stats/stats-chart-tabs/index.jsx]: No way, this test has just launched!

Comment (artpi) [inline client/my-sites/stats/stats-chart-tabs/index.jsx]: I know we are changing our approach to testing, but lets let current test finish ok?

**Comment by rralian** [comment]
Tested and works for me. I understand the hesitance to remove a test, but if it's not getting enough traffic to have significant results or drive revenue, then it's not really worth keeping it active. This looks like a good base for running nudges as a single test. 🚢

**Pull request merged**